.github/workflows/congratulate.yml

```diff
@@ -0,0 +1,23 @@
+name: Congratulate Contributors
+
+on:
+  # it's usually not recommended to use pull_request_target
+  # but we consider it's safe here if we keep the same steps
+  # see: https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+  # and: https://github.com/facebook/react-native/pull/34370/files
+  pull_request_target:
+    types: [closed]
+permissions:
+  pull-requests: write
+jobs:
```

Comment:
Could we add this as a step in utils instead of a dedicated workflow? It will show up as a new step in CI otherwise which would be a bit strange

Reply:
I ran multiple tests and unfortunately we can’t add the congratulations bot as a new step in CI. Basically as of right now, CI is being triggered for the following types of activity:
* opened: Triggered when a pull request is opened.
* synchronize: Triggered when a commit is added to the pull request.
* reopened: Triggered when a pull request is reopened.

The bot needs to be triggered on the **closed** event.
We don’t want to add closed for the existing danger file as it wouldn’t make any sense + we don’t want the bot triggered on the 3 other events. 
So what we could do to avoid having another workflow is add another job to CI (danger + congratulate). Making sure danger is triggered for 3 events and congratulate for closed only. Basically the job would be skipped for event types that they don’t correspond to. This would allow us to have one workflow but with 2 different jobs. Let me know if you would rather that or have two separate workflows?

Reply:
Yes I think that would be best! Thanks